components/operator/hack/ci/Makefile

```diff
@@ -101,17 +77,17 @@ provision-gardener:
 deprovision-gardener:
 	@make -C ${PROJECT_COMMON} deprovision-gardener
 
-.PHONY: run-with-lifecycle-manager
-run-with-lifecycle-manager:
-	@make -C ${PROJECT_COMMON} run-with-lifecycle-manager
-
+# used on test-infra in prow/scripts/cluster-integration/kyma-serverless-integration-k3d.sh
```

Comment:
Thats right. We can remove it when [this](https://github.com/kyma-project/test-infra/pull/9804) PR will be merged.